docs/data-management.rst

```diff
@@ -226,6 +232,87 @@ Typically an administrator works on behalf of other users in a group where the a
+
+-  Plugin for duplication on Command Line:
+
+   - `omero-cli-duplicate on PyPI <https://pypi.org/project/omero-cli-duplicate/#description>`_
+
+
+Setup
+-----
+
+**Duplicate plugin installation**
+
+- Go to the environment where you installed your OMERO.cli as specified under -  https://docs.openmicroscopy.org/latest/omero/users/cli/installation.html.
+
+- Activate the virtual environment.
+
+- Run::
+    
+   $ pip install omero-cli-duplicate
+
+Step-by-Step
+------------
+
+#. On your local machine, open a terminal
+
+#. Activate the virtual environment where ``omero-py`` is installed or add it to ``PATH`` e.g.::
+    
+   $ export PATH=/opt/omero/server/venv3/bin:$PATH
+
+#. The variables ``$ID​1`` and ``$ID2`` below are the IDs of the ​selected Datasets. To duplicate two Datasets with their Images and annotations on both the Images and the Datasets, run::
+    
+   $ ​omero duplicate Dataset:$ID1,$ID2 --report
+
+#. The duplicated Datasets will not be linked to any Project, even if the originals were linked to some Project. 
+
+#. Duplicate two Images with many ROIs on them. The ROIs duplication might take a long time. To exclude the duplication of the ROIs, run::
+
+   $ omero duplicate Image:$ID1,$ID2 --ignore-classes=Roi --report 
+
+#. Find the duplicated Images in the Orphaned Images and Drag and Drop them into a Dataset or create a new Dataset for them.
+
+#. Dupliacte two Projects of another user in read-annotate group type. The group name in our example below is `read-annotate-group`. Log in as `user-1` and duplicate Projects belonging to `user-2` in that group. Further, instead of duplicating all the annotations of `user-2`, link all the original annotations such as Tags, Key-Value pairs and FileAnnotations of `user-2` to the corresponding objects in the duplicate, and duplicate only Comments and Ratings. The duplicate and Comments and Ratings on it will belong to `user-1`, but the other annotations will still belong to `user-2`. Run::
+
+   $ omero logout
+   $ omero login -u user-1 -g read-annotate-group
+   $ omero duplicate Project:$ID1,$ID2 --reference-classes=Annotation --duplicate-classes=CommentAnnotation,LongAnnotation --report
+
+#. Dupliacte two Projects of another user in read-only group type. The group name in our example below is `read-only-group`. If the duplicator is not an administrator, administraotr with restricted privileges or group owner, they cannot link the annotations of another user to their duplicate in read-only group. They might duplicate all the annotations or exclude the duplication of all the annotations by excluding the Link duplication to the relevant objects as shown below. Run::
+
+   $ omero logout
+   $ omero login -u user-1 -g read-only-group
+   $ omero duplicate Dataset:$ID1,$ID2 --ignore-classes=DatasetAnnotationLink,ImageAnnotationLink,Roi --report
```

Comment:
```suggestion
   $ omero duplicate Dataset:$ID1,$ID2 --ignore-classes=IAnnotationLink,Roi --report
```
probably also works if that's more attractive?

Reply:
It is shorter, but much harder to understand. I actually do not understand whether the result of the duplication will be exaclty the same as in the ``classes=DatasetAnnotationLink,ImageAnnotationLink`` or is it a slightly different behaviour (and result) which will also work (not error) which you are suggesting here ?

Reply:
Looking again on what the text above is saying, I think I understand now, it is readable and simpler like you suggest, thanks.